tests/phases/install/cable_install/test_array_install.py

```diff
@@ -12,7 +12,6 @@
 
 import pandas as pd
 import pytest
-
```

Comment:
Not sure why this line was removed, seems like something might be off in isort, though I'm not sure. I think ORBIT and tests should be listed as known first party libraries to avoid this.

Reply:
I did not make these changes. It seems like whoever did, just got rid of empty lines. I can go back and clean it up for readability.

Reply:
Coming back to this. I'm not entirely sure of a solution yet, but you're probably right, there is an .isort.cfg file that lists know_first_party=ORBIT, tests,library. There might be some pre-commit config that isn't set up properly. @RHammond2, do you want to create an issue for it?